### PR TITLE
First draft of gmdf as a separate schema

### PR DESCRIPTION
**Pull request by emackey**
Creating following the split from https://github.com/KhronosGroup/glTF/pull/1416.

/cc @abwood @gbeatty

### CODE REVIEW

**Comment by emackey** [comment]
Here's a [link to the branch](https://github.com/AnalyticalGraphicsInc/gmdf/tree/draft) for easier reading of the main readme.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,82 @@
...
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec, with the `AGI_articulations` and `AGI_stk_metadata` extensions applied.
+
+## Overview
+
+The `AGI_articulations` and `AGI_stk_metadata` glTF 2.0 vendor extensions supply a number of different kinds of metadata to help with analysis of physical objects represented by glTF models.  As an alternative to supplying this metadata as extensions within a glTF file, the same metadata can also be supplied in an external file with the `*.gmdf` file extension, with the same path and base filename as the associated `*.gltf` or `*.glb` file.  One purpose of this is to allow the model author to iterate on their model, re-exporting it from a DCC tool or content pipeline to produce new versions of the model without overwriting the existing metadata.  Optionally, once the model is finalized, the metadata can then be moved into the glTF's JSON structure as extensions, and the external file deleted.
```
Comment (abwood) [inline README.md]: DCC tool?  Dsomething content creation?  Please expand on this acronym.  Also, maybe we call out that gmdf is short for "glTF Meta Data File".

**Review by abwood — APPROVED** [review]
Minor comment.  When you're good let me know and I'll merge this in.  Thanks!

**Comment by abwood** [comment]
Oh, and I guess that once the glTF extensions are merged we can link to them here and vice versa.

File: README.md (hunk 1 above)
Comment (emackey) [reply README.md]: Digital Content Creation.  I'll change it.

**Comment by emackey** [comment]
Updated.

**Pull request merged**